### PR TITLE
Delta Additional Modification

### PR DESCRIPTION
**Pull request by georgedias**
Implement proposed modification reviewed by Emily on 12-Dec-2024
- [x] Add an WHAT and WHY in the Readme.md page
- [x] Introduce the SAF CLI in page 03.md (Delta Process CLI Tools)
- [x] Add light and dark diagrams (page 07)

### CODE REVIEW

File: .github/workflows/deploy-docs.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ name: Deploy Docs
 on:
   push:
     branches:
-      - main
```
Comment (aaronlippold) [inline .github/workflows/deploy-docs.yml:6]: This should not have to change as PRs automatically get deploy previews

Comment (georgedias) [reply .github/workflows/deploy-docs.yml]: using for testing only - for some reason the theme configuration is not working properly when switching from light to dark mode, both images are displayed (see page 7) locally it works as expected.

Comment (Amndeep7) [reply .github/workflows/deploy-docs.yml]: https://github.com/mitre/saf-training/pull/304

Was told to share this with you.  Upstream continues to make breaking changes.  You might want to see if changing where the setting was placed fixes your issue that you're having.

File: src/courses/delta/README.md [hunk 2]
```diff
@@ -19,3 +19,12 @@ By the end of this class, you should be able to:
 - Use the delta fuzzy matching system to map controls between two profiles
 - Identify and review delta process history with generated logs and artifacts
 - Format InSpec profile controls appropriately before and/or after executing delta commands
+
+### 1.3 Delta Demystify
+Cyber security, the practice of protecting computer systems, networks, and data is an ongoing effort
+that demands vulnerability vigilance (current and future) trough continuous monitoring and notification.
+
+To effectively established an ongoing monitoring process, we need to know what to check for, and
+update the content being checked as vulnerabilities are identified. The MITRE training class [Beginner Security Automation Developer Class](https://deploy-preview-256--mitre-saf-training.netlify.app/courses/beginner/) provides an insight on how to use InSpec Profiles (a collection of automated tests) to conduct testes based on some security guidances.
```
[the remark this answers is not in the document]
Comment (georgedias) [reply src/courses/delta/README.md:28]: fixed

**Review by aaronlippold — APPROVED** [review]

**Pull request merged**